.github/workflows/build-and-push-release-candidate.yml

```diff
@@ -1,35 +1,35 @@
-name: Build and publish docker release candidates from tagged releases
+name: web client - build/pub candidate
```

Comment:
don't need to specify `web-cient` here. I only did that in the classifier repo to distinguish names for the two distinct subprojects/docker-images managed in that one repo